rebar.config

```diff
@@ -3,6 +3,7 @@
 	{jiffy, "0.15.2"},
 	{fusco, "0.1.0"},
 	{cowboy, "2.7.0"},
+	{esqlite, "0.4.0"},
 	{sqlite3, "1.1.6"},
```

Comment:
We do not need to specify `sqlite3` here anymore I think?

Reply:
Yep, forgot to remove it 👍 3c87e86